lib/src/native_updater.dart

```diff
@@ -10,16 +10,16 @@ import 'error_material_alert.dart';
 import 'update_cupertino_alert.dart';
 
 class NativeUpdater {
-  BuildContext _context;
-  bool _forceUpdate;
-  String _appName;
-  String _appStoreUrl;
-  String _playStoreUrl;
-  String _iOSDescription;
-  String _iOSUpdateButtonLabel;
-  String _iOSCloseButtonLabel;
-  String _iOSIgnoreButtonLabel;
-  String _iOSAlertTitle;
+  late BuildContext _context;
+  late bool _forceUpdate;
+  late String _appName;
+  String? _appStoreUrl;
```

Comment:
since an app can provide either of the android/ios setting hence these attributes can be null. Hence making them optional is a better choice.